seatunnel-connectors-v2/README.md

```diff
@@ -49,14 +49,19 @@ own connectors, you need to follow the steps below.
 
 3.Create two packages corresponding to source and sink
 
-​    package org.apache.seatunnel.connectors.seatunnel.{connector name}}.source
-
-​    package org.apache.seatunnel.connectors.seatunnel.{connector name}}.sink
+    package org.apache.seatunnel.connectors.seatunnel.{connector name}}.source
+    package org.apache.seatunnel.connectors.seatunnel.{connector name}}.sink
 
 4.add connector info to plugin-mapping.properties file in seatunnel root path.
 
 5.add connector dependency to seatunnel-dist/pom.xml, so the connector jar can be find in binary package.
 
+6.There are several classes that must be implemented on the source side, namely {connector name} Source, {connector name} SourceFactory, {connector name} SourceReader. Please refer to other connectors for details
+
+7.{Connector Name} SourceFactory needs to be annotated with the **@AutoService (Factory.class)** annotation on the class name, and in addition to the required methods, an additional 'creatSource' method needs to be rewritten
+
+8.{Connector Name} Source needs to override the getProducedCatalogTables method
```

Comment:
Please add sink too

Reply:
But I have only done source development and not sink development, so I am not sure if what I am writing is correct